istresearch/rapidpro
# P4-1819 - Add kakao postmaster support

# For the Reviewer

When this is complete, you should approve the PR via github.


## Summary
Add Kakao postmaster support, added chat modes to settings_common, added new icon font for custom channels, fixed pip dependencies. 

#### Release Note
Add Kakao postmaster support, added chat modes to settings_common, added new icon font for custom channels. 

#### Breaking Changes
None.

## Quality Assurance

You have gathered the following items:

Given all of the items above, you have updated your Application ORT at the following locations:
- **Features and Alerting**: <link to app ORT>Required.
- **P4 Alerting**: <link to p4-alerting PR>Required.

## Testing and Verification

1. Using an engage 4.0.6-dev stack, create a postmaster kakao channel
2. Create a contact with a postmaster kakao urn
![Screen Shot 2020-12-09 at 8 58 00 PM](https://user-images.githubusercontent.com/25488234/101711274-4479be00-3a61-11eb-83a2-bb02355fd44b.png)
3. Ensure you can send an outgoing message to the contact
![Screen Shot 2020-12-09 at 8 58 35 PM](https://user-images.githubusercontent.com/25488234/101711309-552a3400-3a61-11eb-9e76-46b4f2154153.png)
![Screen Shot 2020-12-09 at 8 59 24 PM](https://user-images.githubusercontent.com/25488234/101711392-7428c600-3a61-11eb-91e6-6f68401ad9ea.png)
4. Ensure you can receive messages from the contact 
![Screen Shot 2020-12-09 at 9 00 10 PM](https://user-images.githubusercontent.com/25488234/101711433-8acf1d00-3a61-11eb-9a39-9e38ab3805f1.png)
